### PR TITLE
implement --force option for --inject-as-head to allow replacing a curre...

### PR DESCRIPTION
**Pull request by domsj**
...nt corrupt head.db

add different exit codes depending on which database is corrupted

### CODE REVIEW

**Comment by domsj** [comment]
FYI this is related to http://jira.incubaid.com/browse/ARAKOON-438
CI agrees with these changes

File: src/node/dump_store.ml [hunk 1]
```diff
@@ -46,7 +46,9 @@ let dump_store filename =
   Lwt_main.run (t());
   0
 
-let inject_as_head fn node_id cfg_fn =
+exception ExitWithCode of int;;
```
Comment (toolslive) [inline src/node/dump_store.ml:49]: ;; ?

**Comment by toolslive** [comment]
LGTM

File: src/node/dump_store.ml (hunk 1 above)
Comment (domsj) [inline src/node/dump_store.ml:49]: That's not needed indeed, not really sure why I put that there.
I'll just merge it in as is though...

**Pull request merged**